ui/packages/components/package.json

```diff
@@ -48,6 +48,7 @@
     "react-dom": "^18.2.0",
     "react-use": "^17.4.0",
     "react-virtual": "^2.10.4",
+    "recharts": "^2.4.3",
```

Comment:
We should refactor the mini charts to use echarts instead, but it's outside of scope for this one